utils.h

```diff
@@ -53,6 +53,7 @@
 #define STATIC_ASSERT(cond) \
 	static uint8_t CONCATENATE(_cmus_unused_, __LINE__)[2*(cond) - 1] UNUSED
 
+#define CLEANUP(f) __attribute__((cleanup(f)))
```

Comment:
In general, yes. However, the mpris.c file will only be compiled on platforms which have the libsystemd library. This library uses the cleanup attribute extensively internally and sd-bus is designed to be used with this attribute. Hence I think that using it inside mpris.c is appropriate.